script/build.ts

```diff
@@ -118,8 +118,8 @@ async function writeModeOutput(type: string, modes: ReadonlyArray<ModeData>): Pr
 async function writeScssOutput(mode: ModeData): Promise<void> {
   const vars = flattenVars(mode.vars)
 
-  let output = `@mixin primer-${mode.type}-${mode.name}($sel) {\n`
-  output += "  #{$sel} {\n"
+  let output = `@mixin primer-${mode.type}-${mode.name} {\n`
+  output += "  & {\n"
```

Comment:
@simurai Why didn't we think of this when we were first trying to get this to work? 🤔